### ISSUE TITLE
How to change tab separated data using ora2pg to comma separated or something else

### ISSUE THREAD

**Issue by balajithriller** [darold/ora2pg#1536]
Currently in ora2pg data is extracted with tab separated and if i have values in the desc column of a table emp with a tab then there will be error, instead i want to uniquely separate the columns like ||*| or something like that.

**Comment by darold** [comment]
Yes, Ora2Pg use the COPY output format with a tabulation as separator but all tab characters in your data will be escaped into `\t` so no error should be reported.

What issue are you facing exactly?
